droidlet/lowlevel/hello_robot/hello_robot_mover.py

```diff
@@ -146,6 +137,20 @@ def set_look(self, pan_rad, tilt_rad, turn_base=True, world=False):
             dpan = angle_diff(base_pan + self.get_pan(), pan_rad)
         return self.relative_pan_tilt(dpan, dtilt, turn_base=turn_base)
 
+    def is_obstacle_in_front(self, return_viz=False):
+        ret = safe_call(self.cam.is_obstacle_in_front, return_viz)
```

Comment:
where is the `self.cam.is_obstacle_in_front` method defined? can't seem to find it.